src/components/header.tsx

```diff
@@ -29,8 +29,10 @@ export default function Header() {
         <ul className="flex w-2/3 gap-8 p-0">
           <NavLink title="Start" href="/" />
           <NavLink title="Unsere Schule" href="/unsere-schule">
+            <NavLink title="Vorstellung" href="/unsere-schule/vorstellung" />
```

Comment:
Thanks for adding the nav links! The links to "Vorstellung" and "Digitale Schule" aren't related to this feature though. I would suggest adding those links in separate PRs when their respective pages are being added and removing them from this PR for now.

Reply:
True